### PR TITLE
Tweak gatsby deployment instructions

### PR DESCRIPTION
**Pull request by fsteeg**
Based on my experience deploying https://github.com/hbz/lobid/pull/424

### CODE REVIEW

File: gatsby/lobid/README.md [hunk 1]
```diff
@@ -47,14 +55,20 @@ Make sure `node --version` is at least `v10.18.1` and `npm --version` is at leas
 
    Your site is now running at `http://localhost:8000`!
 
-   *Note*: As the port 8000 is closed on emphytos use the proxy http://gatsbydev.lobid.org/. You'll also see a second link: _`http://localhost:8000/___graphql`_, adjust this to the proxy. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).
-   Also, only the dynamically generated pages (this defined in _src/pages_) are served properly.
+   *Note*: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).
```
Comment (dr0i) [inline gatsby/lobid/README.md:58]: How did you use this URL? We have no graphical login to start a browser, so we need some kind of proxy, no?

File: gatsby/lobid/README.md [hunk 2]
```diff
@@ -47,14 +55,20 @@ Make sure `node --version` is at least `v10.18.1` and `npm --version` is at leas
...
+   Also, only the dynamically generated pages (this defined in _src/pages_) are served properly with `gatsby develop`.
 
    Open the `lobid` directory in your code editor of choice and edit e.g. `src/pages/team-de.js`. Save your changes and the browser will update in real time!
 
+2. **Stage**
 
-2. **Deploy**
+   The following assumes being in the web home on emphytos: `ssh <user>@emphytos`; `sudo su`; `cd /srv/www/htdocs/`.
```
Comment (dr0i) [inline gatsby/lobid/README.md:65]: Is this really needed, becoming root?

File: gatsby/lobid/README.md [hunk 3]
```diff
@@ -33,10 +33,18 @@ Make sure `node --version` is at least `v10.18.1` and `npm --version` is at leas
...
    ```shell
-   cd stage.lobid.org/gatsby/lobid
+   git clone https://github.com/hbz/lobid.git
    ```
+   
+   Go to `gatsby/lobid` in the repo:
+   
+   ```shell
+   cd lobid/gatsby/lobid
+   ```
```
Comment (dr0i) [inline gatsby/lobid/README.md:46]: this seems to be production, not staging

**Review by dr0i — COMMENTED** [review]
I am not sure if you did the staging process. See comments inline.

File: gatsby/lobid/README.md (hunk 2 above)
Comment (fsteeg) [reply gatsby/lobid/README.md]: As discussed in standup, I verified it works as user `lobid` and updated the README.

**Review by dr0i — APPROVED** [review]

**Pull request merged**